packages/RangeFilter/src/index.tsx

```diff
@@ -96,7 +96,7 @@ const RangeFilter: FunctionComponent<IRangeFilterProps> = ({
         step={step}
         min-value={minValue}
         max-value={maxValue}
-        className="filter"
+        className={`${name}Range filter`}
```

Comment:
I would maybe remove the whitespace `${name}RangeFilter`

Reply:
we should try to respect BEM convention here